src/main/kotlin/com/ampnet/projectservice/controller/pojo/request/ProjectRequests.kt

```diff
@@ -42,13 +42,24 @@ data class ProjectUpdateRequest(
     val description: String? = null,
     val location: ProjectLocationRequest? = null,
     val roi: ProjectRoiRequest? = null,
+    val startDate: ZonedDateTime? = null,
+    val endDate: ZonedDateTime? = null,
+    @field:Positive
+    val expectedFunding: Long? = null,
+    val currency: Currency? = null,
+    @field:Positive
+    val minPerUser: Long? = null,
+    @field:Positive
+    val maxPerUser: Long? = null,
     val active: Boolean? = null,
     val tags: List<String>? = null,
     val news: List<String>? = null,
     val shortDescription: String? = null
 ) {
     override fun toString(): String {
-        return "ProjectUpdateRequest(name=$name, location=$location, roi=$roi, active=$active, tags=$tags, " +
-            "news=$news, shortDescription=$shortDescription)"
+        return "ProjectUpdateRequest(name=$name, description=$description, location=$location, roi=$roi, " +
```

Comment:
Remove `description` because it can a long text without any meaning to the logs.